cell/cell.go

```diff
@@ -88,9 +90,23 @@ func Underline() Option {
 	})
 }
 
-// Strikethrough strikes through the cell's text. Currently a no-op until tcell is updated to >= v2.0.0
+// Strikethrough strikes through the cell's text. Only works when using the tcell backend.
 func Strikethrough() Option {
 	return option(func(co *Options) {
 		co.Strikethrough = true
 	})
 }
+
+// Inverse inverts the colors of the cell's text.
+func Inverse() Option {
```

Comment:
These lines can be covered by either adding or expanding an existing test case here:

https://github.com/mum4k/termdash/blob/add3cff47493d2be40e164c81fc63a00b150234c/cell/cell_test.go#L76-L90